proto-lens-protoc/app/Data/ProtoLens/Compiler/Generate.hs

```diff
@@ -904,6 +908,8 @@ messageInstance :: Env RdrNameStr -> T.Text -> MessageInfo OccNameStr -> [RawIns
 messageInstance env protoName m =
     [ funBind "messageName" $ match [wildP] $
           var "Data.Text.pack" @@ string (T.unpack protoName)
+    , funBind "messageDescriptor" $ match [wildP] $
```

Comment:
On second thought, maybe we should call this `packedMessageDescriptor` since it returns a ByteString?  That would avoid a name clash if we want to define a function returning the descriptor itself (see my comment about proto-lens-protobuf-types).